curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/counting-cards.md

```diff
@@ -9,7 +9,7 @@ dashedName: counting-cards
 
 # --description--
 
-In the casino game Blackjack, a player can gain an advantage over the house by keeping track of the relative number of high and low cards remaining in the deck. This is called [Card Counting](https://en.wikipedia.org/wiki/Card_counting).
+In the casino game Blackjack, a player can determine whether he has an advantage on the next hand over the house by keeping track of the relative number of high and low cards remaining in the deck. This is called Card Counting.
```

Comment:
I think I prefer the more gender neutral original wording here.

Reply:
Yeab, slipped from me, I will change it